src/iexamine.cpp

```diff
@@ -73,7 +73,7 @@ void iexamine::gaspump(player *p, map *m, const tripoint &examp)
                 item spill( item_it->type->id, calendar::turn );
                 const auto min = item_it->liquid_charges( 1 );
                 ///\EFFECT_DEX decreases amount of gas spilled from a pump
-                const auto max = item_it->liquid_charges( 1 ) * 8.0 / p->dex_cur;
+                const auto max = item_it->liquid_charges( 1 ) * 8.0 / std::max( 1, p->dex_cur );
```

Comment:
@mugling I'm trying to do:
```C++
auto items = g->m.i_at( examp );
for( item item_it : items ) {
  ....
if( item_it.charges < 1 ) {
   items.erase( item_it );
}
```
but the compiler is throwing an error on items.erase. I think it's because item_it isn't an iterator, but unsure how to fix.